core/src/main/kotlin/entity/channel/VoiceChannel.kt

```diff
@@ -23,7 +25,7 @@ public class VoiceChannel(
     override val data: ChannelData,
     override val kord: Kord,
     override val supplier: EntitySupplier = kord.defaultSupplier
-) : CategorizableChannel, VoiceChannelBehavior {
+) : CategorizableChannel, TopGuildMessageChannel, VoiceChannelBehavior {
```

Comment:
```suggestion
) : TopGuildMessageChannel, VoiceChannelBehavior {
```
`TopGuildMessageChannel` is already a `CategorizableChannel`